libs/features/mas/docs/plans.html

```diff
@@ -48,6 +48,9 @@ <h2 id="crd-mrch-plan-ind">Regular Plans Individual Cards <a class="header-ancho
         <a target="_blank" href="https://main--mas--adobecom.hlx.live/studio.html?#path=nala&query=616273eb-3aad-462a-a6d7-6f6857973b77"> Open in Studio</a>
       </div>
   </div>
+  <!-- <div id="plans-collection">
```

Comment:
commented on purpose?
you can already set it without milolibs, its ok that it wont work till merged
```
<a target="_blank" href="https://mwpw-165478--cc--adobecom.hlx.page/drafts/commerce/plans">Plans collection page ↗</a>
 
```